### PR TITLE
feat: add province region explanation

### PR DESCRIPTION
**Pull request by ioane-bendingspoons**
## Description

This PR tackles:

- Add explanation dialog for Province/Region

In particular, the ...

## Checklist

- [x] I have followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md).
- [x] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [x] I have written new tests for my core changes, as applicable.
- [x] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

## Fixes

- Fixes #ISSUE_NUMBER

### CODE REVIEW

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰
- The following files have been modified: build.gradle.


⚠️ Some configuration files don't match the master branch. If you did not perform these changes, **please rebase on the master branch**.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #3204 (id: 7079d18e-a301-4e72-8c6c-eee79b865aae)
- Last verified commit: d08ec280bbf682b33cbbcd60ae66d4306f7a715e
- Time of check: 24/06/2020, 11:17:42 UTC
- The following protected files have been checked for changes: .editorconfig, .gitmodules, Gemfile, Gemfile.lock, app/build.gradle, app/proguard-rules.pro, build.gradle, ci/changelog\_config.tpl, ci/danger/commitlint/index.ts, ci/danger/commitlint/rules.ts, ci/danger/danger.ts, ci/danger/ktlint.ts, dangerfile.ts, fastlane/Appfile, fastlane/Fastfile, gradle.properties, gradlew, gradlew.bat, package.json, publiccode.yml, scheduler\_config.json, settings.gradle, template.properties, yarn.lock.

**Comment by ImmuniDGR** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: Commitlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Thank you for submitting a pull request! The team will review your submission as soon as possible.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d15650514963e36d681b7d4aa15fb9b02157d715
</p>

File: app/src/main/java/it/ministerodellasalute/immuni/ui/onboarding/fragments/viewpager/ProvinceFragment.kt [hunk 1]
```diff
@@ -53,6 +55,7 @@ class ProvinceFragment :
             pageTitle?.alpha = 1 - ratio
             description?.alpha = 1 - ratio
             toolbarTitle?.alpha = ratio
+            knowMore?.alpha = 1 - ratio
```
Comment (marcouberti) [inline app/src/main/java/it/ministerodellasalute/immuni/ui/onboarding/fragments/viewpager/ProvinceFragment.kt:58]: @ioane-bendingspoons the text view should disappear on scrolling. Fixed.

File: app/src/main/java/it/ministerodellasalute/immuni/ui/onboarding/fragments/viewpager/RegionFragment.kt [hunk 2]
```diff
@@ -56,6 +58,7 @@ class RegionFragment :
             pageTitle?.alpha = 1 - ratio
             description?.alpha = 1 - ratio
             toolbarTitle?.alpha = ratio
+            knowMore?.alpha = 1 - ratio
```
Comment (marcouberti) [inline app/src/main/java/it/ministerodellasalute/immuni/ui/onboarding/fragments/viewpager/RegionFragment.kt:61]: @ioane-bendingspoons the text view should disappear on scrolling. Fixed.

**Pull request merged**